app/jobs/course/statistics/assessments_score_summary_download_job.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+class Course::Statistics::AssessmentsScoreSummaryDownloadJob < ApplicationJob
+  include TrackableJob
+  queue_as :lowest
+
+  protected
+
+  def perform_tracked(course, assessment_ids)
+    file_name = "#{Pathname.normalize_filename(course.title)}_score_summary_#{Time.now.strftime '%Y-%m-%d %H:%M'}.csv"
```

Comment:
can just underscore the datetimestamp `%Y%m%d_%H%M`  
otherwise we get a weirdly formatted filename like this: `title of course_score_summary_2024-11-18 16_56.csv`